### PR TITLE
Create close stale issues action

### PR DESCRIPTION
**Pull request by bajanam**

### CODE REVIEW

File: .github/workflows/close_stale_issues.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'There has not been any recent activity on this ticket, so we are marking it as stale. Please comment if you have any additional questions.'
+          days-before-issue-stale: 14
+          days-before-pr-stale: -1
+          days-before-close: 1
```
Comment (kmahar) [inline .github/workflows/close_stale_issues.yml:17]: can we make the window here a little longer so people have more of a window to follow up, like 1 week?

File: .github/workflows/close_stale_issues.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+permissions:
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'There has not been any recent activity on this ticket, so we are marking it as stale. Please comment if you have any additional questions.'
```
Comment (kmahar) [inline .github/workflows/close_stale_issues.yml:14]:
```suggestion
          stale-issue-message: 'There has not been any recent activity on this ticket, so we are marking it as stale. If we do not hear anything further from you, this issue will be automatically closed in one week.
```

File: .github/workflows/close_stale_issues.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+          days-before-issue-stale: 14
+          days-before-pr-stale: -1
+          days-before-close: 1
+          close-issue-message: 'There has not been any recent activity on this ticket, so we are resolving it. Thanks for reaching out!'
```
Comment (kmahar) [inline .github/workflows/close_stale_issues.yml:18]:
```suggestion
          close-issue-message: 'There has not been any recent activity on this ticket, so we are closing it. Thanks for reaching out and please feel free to filed a new issue if you have further questions.
```

File: .github/workflows/close_stale_issues.yml (hunk 1 above)
Comment (bajanam) [reply .github/workflows/close_stale_issues.yml:17]: I updated it to 7 days for stale and 7 for closing for a total of 14 days to match our jira policy.

**Review by kmahar — APPROVED** [review]
lgtm! thanks

**Pull request merged**